vue-reviews/src/router/index.js

```diff
@@ -5,12 +5,13 @@ import ReviewersList from '@/components/Reviewers'
 import MainConfig from '@/components/MainConfig'
 import CapacityDoc from '@/components/capacity/CapacityDoc'
 import StatsTable from '@/components/StatsTable'
+import KnowledgeSharing from '@/components/knowledgesharing/KnowldedgeSharing'
```

Comment:
Please name file properly : to "KnowledgeSharing" instead of "KnowldedgeSharing". 
Should be done both here and within component itself